2021-05-11 17:47:30.0640 ERROR Something went wrong: AutoMapper.AutoMapperMappingException: Error mapping types.

Mapping types:
Company -> CompanyForUpdateDto
Library.Entities.Models.Company -> Library.Entities.DataTransferObjects.CompanyForUpdateDto

Type Map configuration:
Company -> CompanyForUpdateDto
Library.Entities.Models.Company -> Library.Entities.DataTransferObjects.CompanyForUpdateDto

Destination Member:
Employees

 ---> AutoMapper.AutoMapperMappingException: Missing type map configuration or unsupported mapping.

Mapping types:
Employee -> EmployeeForCreationDto
Library.Entities.Models.Employee -> Library.Entities.DataTransferObjects.EmployeeForCreationDto
   at lambda_method(Closure , Employee , EmployeeForCreationDto , ResolutionContext )
   at lambda_method(Closure , Object , CompanyForUpdateDto , ResolutionContext )
   --- End of inner exception stack trace ---
   at lambda_method(Closure , Object , CompanyForUpdateDto , ResolutionContext )
   at CompanyEmployees.Controllers.CompanyController.UpdateCompanyDataPartially(Guid id, JsonPatchDocument`1 patchDoc) in C:\Users\tulshi\source\repos\CompanyEmployeesApp\CompanyEmployees\Controllers\CompanyController.cs:line 141
   at lambda_method(Closure , Object , Object[] )
   at Microsoft.Extensions.Internal.ObjectMethodExecutor.Execute(Object target, Object[] parameters)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.SyncActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeActionMethodAsync()
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeNextActionFilterAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeFilterPipelineAsync>g__Awaited|19_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Diagnostics.ExceptionHandlerMiddleware.<Invoke>g__Awaited|6_0(ExceptionHandlerMiddleware middleware, HttpContext context, Task task)
2021-05-11 17:47:47.0569 ERROR Something went wrong: AutoMapper.AutoMapperMappingException: Error mapping types.

Mapping types:
Company -> CompanyForUpdateDto
Library.Entities.Models.Company -> Library.Entities.DataTransferObjects.CompanyForUpdateDto

Type Map configuration:
Company -> CompanyForUpdateDto
Library.Entities.Models.Company -> Library.Entities.DataTransferObjects.CompanyForUpdateDto

Destination Member:
Employees

 ---> AutoMapper.AutoMapperMappingException: Missing type map configuration or unsupported mapping.

Mapping types:
Employee -> EmployeeForCreationDto
Library.Entities.Models.Employee -> Library.Entities.DataTransferObjects.EmployeeForCreationDto
   at lambda_method(Closure , Employee , EmployeeForCreationDto , ResolutionContext )
   at lambda_method(Closure , Object , CompanyForUpdateDto , ResolutionContext )
   --- End of inner exception stack trace ---
   at lambda_method(Closure , Object , CompanyForUpdateDto , ResolutionContext )
   at CompanyEmployees.Controllers.CompanyController.UpdateCompanyDataPartially(Guid id, JsonPatchDocument`1 patchDoc) in C:\Users\tulshi\source\repos\CompanyEmployeesApp\CompanyEmployees\Controllers\CompanyController.cs:line 141
   at lambda_method(Closure , Object , Object[] )
   at Microsoft.Extensions.Internal.ObjectMethodExecutor.Execute(Object target, Object[] parameters)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.SyncActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeActionMethodAsync()
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeNextActionFilterAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeFilterPipelineAsync>g__Awaited|19_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Diagnostics.ExceptionHandlerMiddleware.<Invoke>g__Awaited|6_0(ExceptionHandlerMiddleware middleware, HttpContext context, Task task)
2021-05-11 17:48:07.3782 ERROR Something went wrong: AutoMapper.AutoMapperMappingException: Error mapping types.

Mapping types:
Company -> CompanyForUpdateDto
Library.Entities.Models.Company -> Library.Entities.DataTransferObjects.CompanyForUpdateDto

Type Map configuration:
Company -> CompanyForUpdateDto
Library.Entities.Models.Company -> Library.Entities.DataTransferObjects.CompanyForUpdateDto

Destination Member:
Employees

 ---> AutoMapper.AutoMapperMappingException: Missing type map configuration or unsupported mapping.

Mapping types:
Employee -> EmployeeForCreationDto
Library.Entities.Models.Employee -> Library.Entities.DataTransferObjects.EmployeeForCreationDto
   at lambda_method(Closure , Employee , EmployeeForCreationDto , ResolutionContext )
   at lambda_method(Closure , Object , CompanyForUpdateDto , ResolutionContext )
   --- End of inner exception stack trace ---
   at lambda_method(Closure , Object , CompanyForUpdateDto , ResolutionContext )
   at CompanyEmployees.Controllers.CompanyController.UpdateCompanyDataPartially(Guid id, JsonPatchDocument`1 patchDoc) in C:\Users\tulshi\source\repos\CompanyEmployeesApp\CompanyEmployees\Controllers\CompanyController.cs:line 141
   at lambda_method(Closure , Object , Object[] )
   at Microsoft.Extensions.Internal.ObjectMethodExecutor.Execute(Object target, Object[] parameters)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.SyncActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeActionMethodAsync()
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeNextActionFilterAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeFilterPipelineAsync>g__Awaited|19_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Diagnostics.ExceptionHandlerMiddleware.<Invoke>g__Awaited|6_0(ExceptionHandlerMiddleware middleware, HttpContext context, Task task)
2021-05-11 17:49:17.3497 ERROR Something went wrong: AutoMapper.AutoMapperMappingException: Error mapping types.

Mapping types:
Company -> CompanyForUpdateDto
Library.Entities.Models.Company -> Library.Entities.DataTransferObjects.CompanyForUpdateDto

Type Map configuration:
Company -> CompanyForUpdateDto
Library.Entities.Models.Company -> Library.Entities.DataTransferObjects.CompanyForUpdateDto

Destination Member:
Employees

 ---> AutoMapper.AutoMapperMappingException: Missing type map configuration or unsupported mapping.

Mapping types:
Employee -> EmployeeForCreationDto
Library.Entities.Models.Employee -> Library.Entities.DataTransferObjects.EmployeeForCreationDto
   at lambda_method(Closure , Employee , EmployeeForCreationDto , ResolutionContext )
   at lambda_method(Closure , Object , CompanyForUpdateDto , ResolutionContext )
   --- End of inner exception stack trace ---
   at lambda_method(Closure , Object , CompanyForUpdateDto , ResolutionContext )
   at CompanyEmployees.Controllers.CompanyController.UpdateCompanyDataPartially(Guid id, JsonPatchDocument`1 patchDoc) in C:\Users\tulshi\source\repos\CompanyEmployeesApp\CompanyEmployees\Controllers\CompanyController.cs:line 141
   at lambda_method(Closure , Object , Object[] )
   at Microsoft.Extensions.Internal.ObjectMethodExecutor.Execute(Object target, Object[] parameters)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.SyncActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeActionMethodAsync()
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeNextActionFilterAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeFilterPipelineAsync>g__Awaited|19_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Diagnostics.ExceptionHandlerMiddleware.<Invoke>g__Awaited|6_0(ExceptionHandlerMiddleware middleware, HttpContext context, Task task)
